.rubocop.yml

```diff
@@ -0,0 +1,27 @@
+AllCops:
+  Exclude:     
+    - 'db/**/*'     
+    - 'config/**/*'     
+    - 'script/**/*'
+    - 'test/**/*.rb'
+    - '**/*.rake'
+    - 'lib/tasks/auto_annotate_models.rake'
+    - 'bin/**/*'
+    - 'Rakefile'
+    - 'vendor/**/*'
+
+Style/Documentation:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
```

Comment:
not sure you want to do this.

Reply:
seems it's relevant for future Ruby versions